co-simulation/fed/mosaic-infrastructure/src/test/java/org/eclipse/mosaic/fed/infrastructure/ambassador/InfrastructureTimeInterfaceTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2021 Old Dominion University. All rights reserved.
```

Comment:
Just noticed this, should be the Leidos copyright statement not the ODU one?

Reply:
Changed to Leidos copyright